code/game/jobs/job/command/auxiliary/pilot.dm

```diff
@@ -1,20 +0,0 @@
-/datum/job/command/pilot
-	title = JOB_PILOT
-	total_positions = 2
-	spawn_positions = 2
```

Comment:
The type `/datum/job/command/pilot/whiskey` should be set back to be back to 2 slots